vtr_flow/scripts/run_vtr_task.py

```diff
@@ -349,15 +350,15 @@ def run_parallel(args, queued_jobs, run_dirs):
 
 
 def create_run_scripts(args, jobs, run_dirs):
-    """ Create the bash script files for each job run """
+    """Create the bash script files for each job run"""
```

Comment:
Should add more commenting around the "scripts" option.  For example around line 310, "elif args.system == "scripts":" 
you could describe what this argument does.
Same thing for the documentation of the command line option --system (line 160 or so):
The help says "What system to run this on" which isn't very detailed.  Should explain what the --system scripts option does somewhere, likely in the help. Details might be in a comment block.


<!--StartFragment-->    parser.add_argument(        "-system",        choices=["local", "scripts"],        default="local",        help="What system to run the tasks on.",    )<!--EndFragment-->



<!--StartFragment-->    parser.add_argument(        "-system",        choices=["local", "scripts"],        default="local",        help="What system to run the tasks on.",    )<!--EndFragment-->



<!--StartFragment-->    parser.add_argument(        "-system",        choices=["local", "scripts"],        default="local",        help="What system to run the tasks on.",    )<!--EndFragment-->



<!--StartFragment-->    parser.add_argument(        "-system",        choices=["local", "scripts"],        default="local",        help="What system to run the tasks on.",    )<!--EndFragment-->



<!--StartFragment-->    parser.add_argument(        "-system",        choices=["local", "scripts"],        default="local",        help="What system to run the tasks on.",    )<!--EndFragment-->



<!--StartFragment-->    parser.add_argument(        "-system",        choices=["local", "scripts"],        default="local",        help="What system to run the tasks on.",    )<!--EndFragment-->




<!--StartFragment-->    parser.add_argument(        "-system",        choices=["local", "scripts"],        default="local",        help="What system to run the tasks on.",    )<br class="Apple-interchange-newline"><!--EndFragment-->




<!--StartFragment-->    parser.add_argument(        "-system",        choices=["local", "scripts"],        default="local",        help="What system to run the tasks on.",    )<br class="Apple-interchange-newline"><!--EndFragment-->